### PR TITLE
This func should be mutating func

### PR DESCRIPTION
**Pull request by varungakhar**
Screenshot 2021-04-22 at 1 13 27 PM

### CODE REVIEW

**Comment by Andrewangeta** [comment]
What the actual issue here? The screenshot you posted has no context?

**Comment by varungakhar** [comment]
when i am trying to create file , i am getting this error
 request.stripe.files.create(file: document_id, purpose: .identityDocument)

**Comment by Andrewangeta** [comment]
What's the actual error? None of the other functions on any other API route needs to be a `mutating func` so why is this one the exception? I need a clear error as to what's  actually happening first.

**Comment by Andrewangeta** [comment]
I see, it's because the implementation itself is a `mutating func` because it modifies the `headers` property in the `struct` itself but the protocol extension isn't mutating.

File: Sources/StripeKit/Core Resources/Files/FileRoutes.swift [hunk 1]
```diff
@@ -17,7 +17,7 @@ public protocol FileRoutes {
     ///   - purpose: The purpose of the uploaded file. Possible values are `business_icon`, `business_logo`, `customer_signature`, `dispute_evidence`, `identity_document`, `pci_document`, or `tax_document_user_upload`.
     ///   - fileLinkData: Optional parameters to automatically create a file link for the newly created file.
     /// - Returns: A `StripeFile`.
-    func create(file: Data, purpose: StripeFilePurpose, fileLinkData: [String: Any]?) -> EventLoopFuture<StripeFile>
+   mutating func create(file: Data, purpose: StripeFilePurpose, fileLinkData: [String: Any]?) -> EventLoopFuture<StripeFile>
```
Comment (Andrewangeta) [inline Sources/StripeKit/Core Resources/Files/FileRoutes.swift:20]: Small indentation here.

**Review by Andrewangeta — CHANGES_REQUESTED** [review]

**Review by Andrewangeta — APPROVED** [review]

**Pull request merged**